src/picongpu/include/particles/ionization/byCollision/ThomasFermi/ThomasFermi_Impl.hpp

```diff
@@ -75,7 +75,11 @@ namespace ionization
 
         using FrameType =  typename SrcSpecies::FrameType;
 
-        /* specify field to particle interpolation scheme */
+        /** specify field to particle interpolation scheme
+         *
+         * @todo this needs to be done independently/twice if ion species (rho) and electron
+         *       species (ene) are of different shape
```

Comment:
In principle one could add another **todo** here
If one were to use two different atomic species they would currently each do their own density and energy calculations. Yet they would have no knowledge of each other. But of course IPD effects do not care about the species and so it remains a technical detail to be solved on how to include that. :smiley: